module/VuFind/src/VuFind/ILS/Driver/AbstractBase.php

```diff
@@ -81,4 +81,28 @@ protected function throwAsIlsException(
     ): void {
         throw new ILSException($msg ?? $exception->getMessage(), 0, $exception);
     }
+
+    /**
+     * Explode a delimited setting to an array
+     *
+     * @param string $value     Setting value
+     * @param bool   $trim      Whether to trim the values
+     * @param string $separator Separator
+     *
+     * @return array
+     */
+    protected function explodeSetting(
```

Comment:
Would it make sense to move this to `\VuFind\Config\ExplodeSettingTrait` so it can also be used outside of ILS drivers? I suspect we're doing this kind of exploding all over the place.

Reply:
Good idea, done.